<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListModelView</class>
 <widget class="QMainWindow" name="ListModelView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>874</width>
    <height>514</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Consolas</family>
    <pointsize>16</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>ListModelView</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QSplitter" name="HS_01">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="GB_01listView">
       <property name="title">
        <string>List View</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QGridLayout" name="GL_01">
          <item row="0" column="1">
           <widget class="QPushButton" name="PB_02clearList">
            <property name="text">
             <string>Clear List</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QCheckBox" name="CB_01listEditable">
            <property name="text">
             <string>List Editable</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="PB_04insert">
            <property name="text">
             <string>Insert Item</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QPushButton" name="PB_05delete">
            <property name="text">
             <string>Delete Item</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="PB_01recovery">
            <property name="text">
             <string>Recover List</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="PB_03append">
            <property name="text">
             <string>Append Item</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="PB_06moveUp">
            <property name="text">
             <string>Move Up</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="PB_07moveDown">
            <property name="text">
             <string>Move Down</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QPushButton" name="PB_08sort">
            <property name="text">
             <string>List Sort</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QListView" name="LV_01"/>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="GB_02plainTextEdit">
       <property name="title">
        <string>Plain Text Edit</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="VL_01">
          <item>
           <widget class="QPushButton" name="PB_09clearText">
            <property name="text">
             <string>Clear Text</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="PB_10showModelStringlist">
            <property name="text">
             <string>Show the stringlist of model</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="PTE_01"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>PB_09clearText</sender>
   <signal>clicked()</signal>
   <receiver>PTE_01</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>672</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>679</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
